Sources/Sentry/include/SentryScreenshot.h

```diff
@@ -13,7 +13,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable NSArray<NSData *> *)appScreenshots;
 
-- (void)saveScreenShots:(NSString *)path;
+/**
+ * Save the current app screen shots in the given directory.
+ * If an app has more than one screen, one image for each screen will be saved.
+ *
+ * @param filesDirectory The path where the images should be saved.
+ */
+- (void)saveScreenShots:(NSString *)filesDirectory;
```

Comment:
`l`: I think `filesDirectory` is not much better than `path`. What about `imageDirectoryPath`?